corehq/apps/app_manager/views/modules.py

```diff
@@ -631,13 +631,22 @@ def edit_report_module(request, domain, app_id, module_id):
     module = app.get_module(module_id)
     assert isinstance(module, ReportModule)
     module.name = params['name']
-    module.report_configs = [ReportAppConfig.wrap(spec) for spec in params['reports']]
+    try:
+        module.report_configs = [ReportAppConfig.wrap(spec) for spec in params['reports']]
+    except Exception:
+        return HttpResponseBadRequest(_("There was a problem processing your request."))
```

Comment:
do we want to consider logging these at all? do you know what category of error you expect to find here?